### ISSUE TITLE
Unwanted longitude conversion 0-360 to -180-180 beginning with CRAN release 1.0.0

### ISSUE THREAD

**Issue by luke-a-rogers** [r-spatial/sf#1751]
tl;dr `st_crop()` and `st_simplify()` unexpectedly converted `sf` objects from longitudes 0-360 to longitudes -180-180 in `sf` versions 1.0.0 to 1.0.2 from CRAN, while version 0.9.8 maintained the original desired 0-360 range of longitudes.

# Set up

I downloaded the Marine Regions World EEZ v11 0 to 360 shapefiles:

  - I navigated to [https://marineregions.org/download_file.php?name=World_EEZ_v11_20191118_HR_0_360.zip](https://marineregions.org/download_file.php?name=World_EEZ_v11_20191118_HR_0_360.zip)
  - I filled out the download form
  - I unzipped the folder to my home directory `~/`

# Unwanted behaviour
The `sf` object started with longitude in the range 0-360, but after the `st_crop()` the longitude was in the range -180-180:

```# Started new session
setwd("~/")
remove.packages("sf")
install.packages("sf")
packageVersion("sf")
#> [1] ‘1.0.2’
library(dplyr)
library(magrittr)
library(sf)
#> Linking to GEOS 3.8.0, GDAL 3.0.4, PROJ 6.3.1

# Read World EEZ shapefile
world_eez <- st_read(
  "~/World_EEZ_v11_20191118_HR_0_360/",
  layer = "eez_v11_0_360",
  as_tibble = TRUE
)
#> ...
#> Simple feature collection with 281 features and 31 fields
#> Geometry type: MULTIPOLYGON
#> Dimension:     XY
#> Bounding box:  xmin: 0 ymin: -85.5625 xmax: 360 ymax: 86.99401
#> Geodetic CRS:  WGS 84

# Isolate Alaska EEZ and remove unneeded variables
alaska_eez <- world_eez %>%
  filter(GEONAME == "United States Exclusive Economic Zone (Alaska)") %>%
  select(geometry)

alaska_eez
#> Simple feature collection with 1 feature and 0 fields
#> Geometry type: MULTIPOLYGON
#> Dimension:     XY
#> Bounding box:  xmin: 167.641 ymin: 47.87644 xmax: 230.0181 ymax: 74.70884
#> Geodetic CRS:  WGS 84
#> # A tibble: 1 × 1
#> geometry
#> <MULTIPOLYGON [°]>
#>   1 (((179.0747 55.34706, 179.1017 55.34267, 179.1037 55.34278, 179.1327 55.34432, 179.1...

# Crop to remove non-Pacific waters
alaska_eez_pac <- alaska_eez %>%
  st_crop(c(xmin = 160, ymin = 45, xmax = 240, ymax = 65.5))

alaska_eez_pac
#> Simple feature collection with 1 feature and 0 fields
#> Geometry type: MULTIPOLYGON
#> Dimension:     XY
#> Bounding box:  xmin: -179.9743 ymin: 49.13073 xmax: 180 ymax: 70.75538
#> Geodetic CRS:  WGS 84
#> # A tibble: 1 × 1
#> geometry
#> *                                                                      <MULTIPOLYGON [°]>
#>   1 (((-159.387 70.75455, -159.3888 70.7515, -159.4096 70.74664, -159.4198 70.74167, -15...
```

# Desired behaviour
An older version (0.9.8) retained the longitude range 0-360 after the `st_crop()`:

```
# Started new session
setwd("~/")
remove.packages("sf")
devtools::install_version("sf", version = "0.9.8", repos = "http://cran.r-project.org")
packageVersion("sf")
#> [1] ‘0.9.8’
library(dplyr)
library(magrittr)
library(sf)
#> Linking to GEOS 3.8.0, GDAL 3.0.4, PROJ 6.3.1

# Read World EEZ shapefile
world_eez <- st_read(
  "~/World_EEZ_v11_20191118_HR_0_360/",
  layer = "eez_v11_0_360",
  as_tibble = TRUE
)
#> ...
#> Simple feature collection with 281 features and 31 fields
#> Geometry type: MULTIPOLYGON
#> Dimension:     XY
#> Bounding box:  xmin: 0 ymin: -85.5625 xmax: 360 ymax: 86.99401
#> Geodetic CRS:  WGS 84

# Isolate Alaska EEZ and remove unneeded variables
alaska_eez <- world_eez %>%
  filter(GEONAME == "United States Exclusive Economic Zone (Alaska)") %>%
  select(geometry)

alaska_eez
#> Simple feature collection with 1 feature and 0 fields
#> Geometry type: MULTIPOLYGON
#> Dimension:     XY
#> Bounding box:  xmin: 167.641 ymin: 47.87644 xmax: 230.0181 ymax: 74.70884
#> Geodetic CRS:  WGS 84
#> # A tibble: 1 × 1
#> geometry
#> <MULTIPOLYGON [°]>
#>   1 (((179.0747 55.34706, 179.1017 55.34267, 179.1037 55.34278, 179.1327 55.34432, 179.1...

# Crop to remove non-Pacific waters
alaska_eez_pac <- alaska_eez %>%
  st_crop(c(xmin = 160, ymin = 45, xmax = 240, ymax = 65.5))

alaska_eez_pac
#> Simple feature collection with 1 feature and 0 fields
#> Geometry type: POLYGON
#> Dimension:     XY
#> Bounding box:  xmin: 167.641 ymin: 47.87644 xmax: 230.0181 ymax: 65.5
#> Geodetic CRS:  WGS 84
#> # A tibble: 1 × 1
#> geometry
#> *                                                                                <POLYGON [°]>
#>   1 ((179.0747 55.34706, 179.1017 55.34267, 179.1037 55.34278, 179.1327 55.34432, 179.1618 55...
```

**Comment by edzer** [comment]
The old behavior is restored with `sf_use_s2(FALSE)`.
